app/main/views/briefs.py

```diff
@@ -81,6 +84,10 @@ def ask_brief_clarification_question(brief_id):
 def start_brief_response(brief_id):
     brief = get_brief(data_api_client, brief_id, allowed_statuses=['live'])
 
+    if not (datetime.strptime(current_app.config['FEATURE_FLAGS_NEW_SUPPLIER_FLOW'], "%Y-%m-%d")
```

Comment:
That view is behind the `@feature.is_active_feature('NEW_SUPPLIER_FLOW')` decorator. If the flag is false, then the route will 404 straight off the bat without going anywhere near this code, so I think it's fine?